cmd/mandump/main.go

```diff
@@ -17,8 +17,9 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/ulikunitz/xz"
-	"go.spiff.io/nxtools/xrepo"
+	"go.spiff.io/mandump/internal/nxtools/xrepo"
```

Comment:
```suggestion
	"go.spiff.io/xmandump/internal/nxtools/xrepo"
```

This should be xmandump. I think the project got renamed early on, so the module name would also have to change. I'll probably fix that in a commit after merging this after work.

Reply:
Ok! For that matter, would you like me to put `nxtools` inside `pkg` instead of `internal`? If they are in internal, I think people can't import it into their own packages.